### DISCUSSION TITLE
problemas con mi cuenta en github

### DISCUSSION THREAD

**Discussion by LeaBertero** [community/community#73289] [New to GitHub]
Buenas tardes comunidad.. Me gustaria que me ayuden, por que tengo problemas con mi cuenta , ya que no deja que los demas usuarios la puedan acceder, me genera un error 404, cuando quieren ingresar desde el elnace. muchas gracias ! necesito recuperar mi cuenta urgente ya que soporte  no me lo piede resolver y no se que hacer, ayuda por favor !

**Comment by jorgearana** [comment]
Estimado LeaBertero, veo que has colocado varios post con problemas con tu cuenta. Desafortunadamente yo también tengo problemas, no me deja agregar usuarios y no encuentro como solucionarlo.

Por lo visto Github no responde ni brinda ningún tipo de solución al respecto y al ver que no te responde nadie de la comunidad, es porque no existe la solución.

bueno, habrá que buscar otra forma de mantener repositorios. ¿Existe otro tipo de servicio parecido a Github? Quizás lo más inteligente sería empezar a buscar a alguien que si te brinde respuestas.

Saludos
